gallery/how_to/work_with_microtvm/micro_custom_ide.py

```diff
@@ -0,0 +1,356 @@
+#
+#    * Note: Make sure to write your code inside the sections marked by USER CODE BEGIN <...> and USER CODE END <...>. The code outside these sections get erased if you regenerate the initialization code.
+#
+#        .. code-block:: c
+#
+#         HAL_GPIO_TogglePin(LD2_GPIO_Port, LD2_Pin);
+#         HAL_UART_Transmit(&hlpuart1, "Hello World.\r\n", 14, 100);
+#         HAL_Delay(1000);
+#
+# #. from the menu bar, select Project -> Build (or right click on project name and select Build). This should build the project and generate the .elf file. Select Run -> Run to download the binary on your MCU. If the "Edit Configuration" window opens, just click "OK".
+#
+# #. Open the terminal console on your host machine. On Mac you can simply use the "screen <usb_device> 115200" command. An LED should blink on the board and the string "Hello World" should print out on your terminal console every second.
+#
+
+######################################################################
+# Import the model to the generated project
+# ------------------------------------------
+#
+# To integrate the compiled model into the generated project, follow these steps:
+#
+# #. Extract the tar file and include it in the project
+#
+#    * If you extract it inside the project folder on your disk, right click on the project name and select "Refresh" and it would appear in the Project Explorer panel.
+#    * Open the project Properties. (by right clicking on the project name and selecting "Properties" or by selecting Project -> Properties from the menu bar).
+#    * Select C/C++ General -> Paths and Symbols. In Source Location tab, click "Add folder" and select the "model" folder.
+#    * If you extract the model file somewhre else, click on the "Link Folder" button, check the box for "Link to folder in the file system" in the window that appears, click "Browse" and select the model folder.
+#
+# #. If you used CMSIS-NN in compiling the model, you need to include the CMSIS-NN source files in your project too.
+#
+#    * Download or clone the files from the `CMSIS-NN repository <https://github.com/ARM-software/CMSIS-NN>`_, and follow the above steps to include the CMSIS-NN folder in the project.
+#
+# #. Open the project properties. In C/C++ build -> Settings: add the following folders to the list of Included Paths for GCC (and G++ if you have a C++ project) by clicking on the "+" button, selecting "Workspace" and navigating to each folder:
+#
+#    * model/runtime/include
+#    * model/codegen/host/include
+#    * CMSIS-NN/include
+#    * model/samples
+#
+# #. Copy crt_config_template.h from model/runtime/template to the Core/Inc folder, and rename it to crt_config.h
+#
+# #. Exclude following folders from build (right click on the folder name, select Resource Configuration → Exclude from build). Check Debug and Release configurations.
+#
+#    * model/runtime/template
```

Comment:
This section should change a bit once we add the platform.c.template to the MLF file. Let's just revisit this section once that PR is merged.

Reply:
I made the changes according to the recent updates in the MLF file.